python/doc/theory/meta_modeling/enumeration_strategy.rst

```diff
@@ -3,42 +3,61 @@
+strategy for enumerating the infinite PC series has to be defined.
+This is the scope of the current section.
 
 Given an input random vector :math:`\vect{X}` with prescribed
 probability density function (PDF) :math:`f_{\vect{X}}(\vect{x})`, it is
-possible to build up a *polynomial chaos* (PC) basis
-:math:`\{\psi_{\idx},\idx \in \NM\}` . Of interest is the definition of
-enumeration strategies for exploring this basis, i.e. of suitable
-*enumeration functions* :math:`\tau` from :math:`\Nset` to :math:`\NM`,
-which creates a one-to-one mapping between an integer :math:`j` and a
-multi-index :math:`\idx`.
+possible to build up a *functional chaos* multivariate basis
+:math:`\{\psi_{\idx},\idx \in \NM\}` by tensorization of multiple univariate
+basis associated to each variable.
 
-Linear enumeration strategy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For that we define the multi-indices associated to the marginal degrees of the
+univariate basis terms:
+
+.. math::
+
+    \vect{\alpha} = (\alpha_1, \dots, \alpha_{n_X}) \in \mathbb{N}^{n_X}
+
+The component :math:`\alpha_i` defines the marginal degree of the univariate
+basis term associated to the variable :math:`X_i`.
+Then, the multivariate basis term :math:`\psi_{\idx}` is defined by the product:
+
+.. math::
+
+    \psi_{\idx} (\xi) = \pi_{\alpha_1}^{(1)}(\xi_1) \times \dots \times \pi_{\alpha_{n_X}}^{({n_X})}(\xi_{n_X}) \forall \xi \in \mathbb{R}^{n_X}
+
+The total degree :math:`|{\idx}|` of the multivariate basis term :math:`\psi_{\idx}`
+is the sum of the marginal degrees of the univariate basis terms:
+
+.. math::
 
-Let us first define the *total degree* of any multi-index :math:`\idx`
-in :math:`\NM` by :math:`\sum_{i=1}^{n_X} \alpha_i`. A natural choice to
-sort the PC basis (i.e. the multi-indices :math:`\idx`) is the
-lexicographical order with a constraint of increasing total degree.
-Mathematically speaking, a bijective enumeration function :math:`\tau`
-is defined by:
+    |{\idx}| = \alpha_1 + \dots + \alpha_{n_X}
+
+Of interest is the definition of enumeration strategies for exploring this basis,
+i.e. of suitable *enumeration functions* :math:`\tau` from :math:`\Nset` to :math:`\NM`,
+which creates a one-to-one mapping between an integer :math:`j` and a multi-index :math:`\idx`.
+
+Mathematically speaking, it is a bijective enumeration function :math:`\tau` defined by:
 
 .. math::
 
    \begin{array}{llcl}
          \tau \, : & \Nset & \longrightarrow & \NM \\
-         &  j & \longmapsto & \{\alpha_1,\dots, \alpha_M\} \, \equiv \, \{\tau_1(j),\dots,\tau_M(j)\} \\
+         &  j & \longmapsto & \{\alpha_1,\dots, \alpha_M\} \, \equiv \, \{\tau_1(j),\dots,\tau_{n_X}(j)\} \\
        \end{array}
 
-such that:
+Linear enumeration strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A natural choice to sort the PC basis (i.e. the multi-indices :math:`\idx`) is the
+lexicographical order with a constraint of increasing total degree, such that:
 
 .. math:: \tau(0) \, \, = \, \, \{0,\dots,0\}
 
```

Comment:
Please split the equations on lines 68 to 76 into separate equations. This will improve (i.e. create) the readability.

Reply:
ok, used your explanations/equations